library/std/src/sys/sgx/abi/usercalls/alloc.rs

```diff
@@ -185,6 +185,12 @@ pub struct UserRef<T: ?Sized>(UnsafeCell<T>);
 #[unstable(feature = "sgx_platform", issue = "56975")]
 pub struct User<T: UserSafe + ?Sized>(NonNull<UserRef<T>>);
 
+#[unstable(feature = "sgx_platform", issue = "56975")]
+unsafe impl<T: UserSafeSized> Send for User<T> {}
```

Comment:
Just checking that it's okay for `User<T>` to be `Send` even if `T` itself is not `Send`.